recipe/build.sh

```diff
@@ -3,54 +3,101 @@
 # unset the SUBDIR variable since it changes the behavior of make here
 unset SUBDIR
 
-USE_NONFREE=no
-
 declare -a _CONFIG_OPTS=()
 
-# We do not care what the defaults are as they could change. Be explicit
-# about every flag.
-if [[ ${USE_NONFREE} == yes ]]; then
-  _CONFIG_OPTS+=("--enable-nonfree")
-  _CONFIG_OPTS+=("--disable-gpl")
-  _CONFIG_OPTS+=("--disable-gnutls")
-  _CONFIG_OPTS+=("--enable-openssl")
-  # The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
-  _CONFIG_OPTS+=("--enable-libopenh264")
-  # GPL-3.0
-  _CONFIG_OPTS+=("--enable-libx264")
+if [[ ${target_platform} == win-64 ]]
+then
+  # the target-os and toolchain picked up by default is mingw, so we have to configure
+  # these flags ourselves to get it to build properly
+  _CONFIG_OPTS+=("--ld=${LD}")
+  _CONFIG_OPTS+=("--target-os=win64")
+  _CONFIG_OPTS+=("--toolchain=msvc")
+  _CONFIG_OPTS+=("--host-cc=${CC}")
+  _CONFIG_OPTS+=("--enable-cross-compile")
+  # ffmpeg by default attempts to link to libm
+  # but that doesn't exist for windows
+  _CONFIG_OPTS+=("--host-extralibs=")
+  # we don't want pthreads on win, it's not posix compliant
```

Comment:
```suggestion
  # The configure script doesn't seem to detect that we are on windows where pthreads is not a thing (without relying on msys2, etc).
  # Force disable pthreads and force enable windows native threads.
```
It's not about compliance, it's about pthreads not existing in the Windows world.

Reply:
well if windows were posix compliant, I guess they would? anyway will include this.